ee/clickhouse/sql/session_recording_events.py

```diff
@@ -90,3 +90,5 @@
 DROP_SESSION_RECORDING_EVENTS_TABLE_SQL = (
     f"DROP TABLE IF EXISTS {SESSION_RECORDING_EVENTS_TABLE} ON CLUSTER {CLICKHOUSE_CLUSTER}"
 )
+
+UPDATE_RECORDINGS_TABLE_TTL_SQL = "ALTER TABLE session_recording_events TTL toDate(created_at) + INTERVAL {weeks} WEEK"
```

Comment:
Use query parameters instead - sql injection is bad even if done by staff users.

`toIntervalWeek(%(weeks)s)`  https://clickhouse.com/docs/en/sql-reference/functions/type-conversion-functions/#function-tointerval